tiledb/sm/array_schema/array_schema.h

```diff
@@ -297,6 +299,28 @@ class ArraySchema {
   /** Returns the array schema version. */
   uint32_t version() const;
 
+  /** Set a timestamp range for the array schema */
```

Comment:
Add a period at the end of each sentence in a block comment like this and others.

Reply:
done, thanks